src/main/resources/db/migration/0200/V232__FacilityInventoryBatchId.sql

```diff
@@ -4,11 +4,11 @@ CREATE OR REPLACE VIEW nursery.facility_inventories AS
 SELECT organization_id,
        species_id,
        facility_id,
-       id                                       AS batch_id,
+       STRING_AGG(CAST(id AS TEXT), ',')        AS batch_ids,
```

Comment:
@sgrimm This is more what I was actually looking for, and doesn't disaggregate the view like what I had before

Reply:
This seems inconsistent with the way we handle lists of associated objects everywhere else in the search API. If you export this view as a CSV, you will get a comma-delimited value whereas other multi-value fields get exported with newline delimiters.

Probably clearest to explain what I mean by example; I will throw together a quick PR to demonstrate.

Reply:
#1582 has the alternate approach to this.

Reply:
Thanks so much